### ISSUE TITLE
playbooks fails when firewalld is not installed on target machine

### ISSUE THREAD

**Issue by alainpham** [nmasse-itix/OpenShift-Implementation-at-ITIX#1]
The switch-to-iptables playbook fails when there is no firewalld installed on the target machine. I'm using the qcow2 cloud image of RHEL 7.5.

Here is the output of the error

TASK [Disable firewalld] ***************************************************************************************************************************************
fatal: [prime]: FAILED! => {"changed": false, "msg": "Could not find the requested service firewalld: host"}
	to retry, use: --limit @/root/ansible/OpenShift-Implementation-at-ITIX/playbooks/switch-to-iptables.retry

**Comment by nmasse-itix** [comment]
Yes, given that the documentation now recommends firewalld, I could just remove this playbook.
https://docs.openshift.com/container-platform/3.9/install_config/install/advanced_install.html#advanced-install-configuring-firewalls

**Comment by nmasse-itix** [comment]
"While iptables is the default firewall, firewalld is recommended for new installations."
